wwwroot/src/pages/Landing.js

```diff
@@ -0,0 +1,260 @@
+  },
+  landing1CopyContainer: {
+    width: theme.spacing(55),
+    margin: 0,
+  },
+  landing1CopyTitle: {
+    fontSize: '40px',
+    lineHeight: '54px',
+    fontWeight: 600,
+    color: theme.palette.primary.main,
+  },
+  landing1CopyBody: {fontSize: '18px'},
+
+  landing1ButtonCollection: {
+    marginTop: theme.spacing(3),
+    justifyContent: 'flex-end',
+  
+    '& a': {
+      width: theme.spacing(20.25),
+      height: theme.spacing(5.25),
+      boxShadow: 'none',
+
+      '&:last-child': {marginLeft: theme.spacing(2)},
+      '& p': {
+        fontSize: '16px', 
+        fontWeight: 700,
+        letterSpacing: '0.05em'
+      }
+    }
+  },
+  landing1TriangleChild: {
+    width: '262px',
+    height: '411.28px',
+    position: 'relative',
+    top: -theme.spacing(56),
+    left: theme.spacing(53),
+    clipPath: 'polygon(50% 22%, 3% 76%, 97% 76%)'
+  },
+  landing1CircleChild: {
+    width: '233px',
+    height: '350px',
+    position: 'relative',
+    top: -theme.spacing(67),
+    left: theme.spacing(52),
+    clipPath: 'circle(116px at 50% 66%)'
+  },
+  landing1SquareChild: {
+    width: '345px',
+    height: '233.09px',
+    position: 'relative',
+    top: -theme.spacing(33.5),
+    left: theme.spacing(2.5),
+    clipPath: 'polygon(11% 0%, 11% 100%, 76% 100%, 76% 0%)'
+  },
+
+  
+  landing2root: {
+    height: '86vh',
+    paddingTop: theme.spacing(6)
+  },
+  landing2Container: {
+    textAlign: 'center',
+    [theme.breakpoints.down('lg')]: {
+      padding: '0 10%'
+    }
+  },
+  landing2Title: {
+    fontSize: '32px',
+    fontWeight: 600,
+    marginBottom: theme.spacing(9)
+  },
+  landing2ItemBox: {
+    width: theme.spacing(48),
+    margin: 0,
+
+    '& h3': {
+      fontSize: '24px',
+      fontWeight: 600,
+      marginBottom: theme.spacing(1)
+    },
+    '& p': {
+      fontSize: '18px',
+      lineHeight: '25px'
+    },
+    '&:last-child > div': {
+      width: theme.spacing(16),
+      position: 'relative',
+      top: -theme.spacing(1)
+    }
+  },
+  landing2ImgContainer: {
+    width: theme.spacing(15),
+    height: theme.spacing(15),
+    margin: `0 auto ${theme.spacing(2.5)}px`
+  },
+  test: {backgroundColor: 'red'}
+}))
+
+
+function Landing1() {
+  const classes = useStyles();
+
+  return (
+    <Container
+      className={classes.landing1root}
+      maxWidth='md'
+    >
+      <Container className={classes.landing1CopyContainer}>
+        <Typography className={classes.landing1CopyTitle} component='h1'>
+          {landing1Data.title}
+        </Typography>
+
+        <Typography className={classes.landing1CopyBody}>
+          {landing1Data.copy}
+        </Typography>
+
+        <Box 
+          className={classes.landing1ButtonCollection} 
+          display='flex'
+          justifyContent="flex-end"
+        >
+          <Button
+            variant='outlined'
+            color='primary'
+            component={RouterLink}
+            to='/why-us'
+          >
+            <Typography>
```

Comment:
the name of the button does not need to be wrapped into Typography, it's extra unnecessary styling. Button provides "label" class for that purpose.

Reply:
done!